packages/circuit-ui/components/Card/Card.tsx

```diff
@@ -35,28 +32,25 @@ export interface CardProps {
   spacing?: 'mega' | 'giga';
 }
 
+// FIXME: Replace border-radius with theme value in v3.
 const baseStyles = ({ theme }: StyleProps) => css`
   label: card;
   background-color: ${theme.colors.white};
-  border-radius: ${theme.borderRadius.mega};
+  border-radius: 16px;
+  border: ${theme.borderWidth.mega} solid ${theme.colors.n200};
   display: flex;
   flex-direction: column;
   justify-content: space-between;
 `;
 
-const shadowStyles = ({ theme, shadow = 'single' }: StyleProps & CardProps) => {
-  if (!shadow) {
-    return null;
+const shadowStyles = ({ shadow }: CardProps) => {
+  if (shadow) {
+    deprecate(
```

Comment:
We should remember to clean up all of these deprecated props in the next major, too 🙂

Reply:
[Yes.](https://sumupteam.atlassian.net/browse/DS-165)